config/routes.rb

```diff
@@ -1,8 +1,18 @@
 Rails.application.routes.draw do
+
+  namespace :api do
+    namespace :v1 do
+      namespace :items do
+        get 'find', to: "find#show", defaults: {format: :json}
+      end
+    end
+  end
+
   namespace :api do
     namespace :v1 do
       resources :merchants, only: [:show, :index], defaults: {format: :json}
       resources :items,     only: [:show, :index], defaults: {format: :json}
     end
   end
+
```

Comment:
when we meet up today let's make sure we are consistent with our spacing at the beginning and end of classes. I think we both have been doing it different ways.